configs/configs.go

```diff
@@ -36,6 +37,8 @@ var (
 
 	// NoOutputTimeout is the timeout after Steps are aborted, when no output is received
 	NoOutputTimeout time.Duration
+
+	LoggerType = log.ConsoleLogger
```

Comment:
Do we need to store the active logger type? Should not we just create the global logger at the beginning and then everything else only wraps that global one and does not need to create new ones.

Reply:
Removed this globa variable, instead `log` package is updated to use a single parameter in the Logger constructor: `LoggerOpts`.
The current global logger options can be get by `log.GetGlobalLoggerOpts` after modifying it you can create a new logger instance easily: 

```
```